packages/bot-web-ui/src/components/load-modal.jsx

```diff
@@ -96,7 +96,9 @@ const Recent = ({
                     })}
                 >
                     <div>
-                        {localize("If you've recently used bots but don't see them in this list. It may because you:")}
+                        {localize(
+                            "If you've recently used bots but don't see them in this list. It may be because you:"
```

Comment:
Could we update this text, as per copywriting, to:

```suggestion
                            "If you've recently used bots but don't see them in this list, it may be because you:"
```

Reply:
Updated.